app/views/adoption_applications/new.html.erb

```diff
@@ -102,7 +104,7 @@
     <%= f.text_area :preferred_characteristics, rows: 6 %>
     
     <div class="Submit">
-        <%= f.submit "Submit Adoption Application"%>
+      <%= f.submit "Submit Adoption Application"%>
```

Comment:
This is a pretty long form. Thoughts on extracting it into a partial called `_form_fields.html.erb` ? That might make this view template a little less daunting.